hedera-node/hedera-app/src/test/java/com/hedera/node/app/services/NodeRewardManagerTest.java

```diff
@@ -104,6 +107,9 @@ class NodeRewardManagerTest {
 
     @BeforeEach
     void setUp() {
+        // to make it work for the multiple node in one JVM case, we need reset the default instance path every time
+        // we create another instance of MerkleDB.
+        MerkleDb.resetDefaultInstancePath();
```

Comment:
@imalygin should these changes be only in `16785-D-virtualmap-keybytes-valueobjects` branch?